GCMtools/utils/interface.py

```diff
@@ -10,27 +10,406 @@
+        Set the data to be used for the interface
+
+        Parameters
+        ----------
+        time: int
+            timestep to be used
+        tag: str
+            tag of the model to be used
+        regrid_lowres: bool, optional
+            Can be useful, if your GCMT uses a very detailed grid
+        """
+        self._set_data_common(time, tag=tag, regrid_lowres=False)
+
+    def _set_data_common(self, time, tag=None, regrid_lowres=False):
+        """
+        Set the data to be used for the interface
+
+        Parameters
+        ----------
+        time: int
+            timestep to be used
+        tag: str
+            tag of the model to be used
+        regrid_lowres: bool, optional
+            Can be useful, if your GCMT uses a very detailed grid
+        """
+        ds = self.gcmt.get_one_model(tag).sel(time=time)
+
+        if regrid_lowres:
+            dlon = 15
+            dlat = 15
+
+            lonb = np.arange(-180, 180 + dlon, dlon)
+            latb = np.arange(-90, 90 + dlat, dlat)
+
+            lons = 0.5 * (lonb[1:] + lonb[:-1])
+            lats = 0.5 * (latb[1:] + latb[:-1])
+
+            ds = ds.interp(lon=lons, lat=lats)
+
+        self.ds = ds
+        self.chemistry.set_data(ds)
+
+    def chem_from_poorman(self, temp_key='T', CO=0.55, FeH=0.0):
+        """
+        Calculate equilibrium abundancies with poorman from pRT
+
+        Parameters
+        ----------
+        temp_key: str, optional
+            The key to the temperature field used for the abundancies.
+            Defaults to T.
+        CO: float, optional
+            The C/O ratio. Currently only one global value allowed. Defaults to 0.55.
+        FeH: float, optional
+            The metalicity ratio. Currently only one global value allowed. Defaults to 0.0.
+        """
+        return self.chemistry.chem_from_poorman(temp_key=temp_key, CO=CO, FeH=FeH)
+
+
+class pRTInterface(Interface):
+    """
+    Interface with petitRADTRANS
+    """
+
+    def __init__(self, gcmt, pRT):
+        """
+        Constructs the Interface and links to pRT.
+
+        Parameters
+        ----------
+        gcmt: GCMTools Object
+            A GCMTools object that is linked to the interface
+        pRT: petitRADTRANS.Radtrans
+            A pRT Radtrans object
+        """
+        super().__init__(gcmt)
+        self.pRT = pRT
+
+    def set_data(self, time, tag=None, regrid_lowres=False):
+        """
+        Set the data to be used for the interface
+
+        Parameters
+        ----------
+        time: int
+            timestep to be used
+        tag: str
+            tag of the model to be used
+        regrid_lowres: bool, optional
+            Can be useful, if your GCMT uses a very detailed grid
+        """
+        self._set_data_common(time, tag=tag, regrid_lowres=False)
+
+        if self.ds.p_unit == 'bar':
+            press = self.ds.Z.values
+        elif self.ds.p_unit == 'Pa':
+            press = self.ds.Z.values/1e5
```

Comment:
Same as above

Reply:
`self._set_data_common` does that.
https://github.com/exorad/GCMtools/blob/31970026ebb3aacf21f87843a6f6861cd7264518/GCMtools/utils/interface.py#L191

Reply:
fixed in new commit